process/project_proposals.adoc

```diff
@@ -36,27 +36,21 @@ image::incubation-process.png[Incubation process]
 
 . *Project Proposal* 
  * Incubation proposed through a https://github.com/cncf/toc/pulls[GitHub pull request]
-. *TOC Triage* _2 weeks for a straightforward referral to a SIG; decline or defer could require more discussion_
-   * Performed by any TOC member
-   * Lazy TOC consensus before declining outright or deferring assessment at this time
-   * Picks appropriate https://github.com/cncf/toc/blob/main/sigs/cncf-sigs.md[CNCF SIG] to perform assessment
-. *SIG Assessment* _1-2 months_
-   * Project presents to SIG
-   * Lightweight - this is pre-DD
-   * Confirms that project accepts the https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc#incubation-stage[Incubation requirements]
-   * SIG makes recommendation to TOC: Sandbox / Incubation / Decline 
-   * Note that this step is still meant to be a lightweight, pre-DD process
-. *TOC Incubation Sponsor*
-   * If a TOC member steps forward as an Incubation Sponsor, due diligence commences
+. *TOC Incubation Sponsor* 
+   * The project cannot move forward to Due Diligence unless and until a TOC member steps forward as an Incubation Sponsor. 
+   * The TOC may agree that the project does not (yet) meet the https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc#incubation-stage[Incubation requirements] and suggest that project apply for Sandbox instead. 
+   * If a TOC Incubation Sponsor has not stepped forward within two months after the proposal PR is submitted, projects may request that their project proposal is discussed at a forthcoming TOC meeting by adding it to the Working Doc. The outcome of this is discussion is either that a sponsor steps forward, or that the TOC votes to admit the project to Sandbox, or the proposal is rejected (projects may reapply after six months). The default outcome is that if a sponsor does not step forward, the proposal is rejected. 
```

Comment:
"The default outcome is that if a sponsor" -> "Even after all those steps if  a sponsor" is this better?